File: python/1131.py
_qtd = _vinter = _vgremio = _empates = 0
while True:
    _qtd += 1
    gols = input()
    if int(gols[0]) > int(gols[2]):
        _vinter += 1
    elif int(gols[0]) == int(gols[2]):
        _empates += 1
    else:
        _vgremio += 1
    while True:
        print('Novo grenal (1-sim 2-nao)')
        opcao = int(input())
        if opcao == 1:
            break
        elif opcao == 2:
            break
    if opcao == 2:
        break
print(_qtd, 'grenais')
print('Inter:',_vinter)
print('Gremio:',_vgremio)
print('Empates:',_empates)
if _vinter == _vgremio:
    print('Nao houve vencedor')
elif _vinter > _vgremio:
    print('Inter venceu mais')
else:
    print('Gremio venceu mais')

File: c++/1131.cpp
#include <iostream>

using namespace std;

int main() {
    int p1, p2, i, op, qtd=1;
    int inter=0, gremio=0, empates=0;

    cin >> p1 >> p2;
    if (p1 == p2) {
        empates += 1;
    } else if (p1 > p2) {
        inter += 1;
    } else {
        gremio += 1;
    }
    cout << "Novo grenal (1-sim 2-nao)\n";
    cin >> op;
    while (op == 1) {
        cin >> p1 >> p2;
        if (p1 == p2) {
            empates += 1;
        } else if (p1 > p2) {
            inter += 1;
        } else {
            gremio += 1;
        }
        qtd += 1;
        cout << "Novo grenal (1-sim 2-nao)\n";
        cin >> op;
    }

    cout << qtd << " grenais" << "\n";
    cout << "Inter:" << inter << "\n";
    cout << "Gremio:" << gremio << "\n";
    cout << "Empates:" << empates << "\n";
    if (inter == gremio) {
        cout << "Nao houve vencedor\n";
    } else if (inter > gremio) {
        cout << "Inter venceu mais\n";
    } else {
        cout << "Gremio venceu mais\n";
    }
    return 0;
}